doc/classes/SceneTree.xml

```diff
@@ -190,8 +189,9 @@
 			<param index="1" name="property" type="String" />
 			<param index="2" name="value" type="Variant" />
 			<description>
-				Sets the given [param property] to [param value] on all members of the given group.
-				[b]Note:[/b] [method set_group] will set the property immediately on all members at once, which can cause stuttering if a property with an expensive setter is set on lots of members.
+				Sets the given [param property] to [param value] on all nodes inside this tree added to the given [param group]. Nodes that do not have the [param property] are ignored. See also [method call_group] and [method notify_group].
+				[b]Note:[/b] This method acts immediately on all selected nodes at once, which may cause stuttering in some performance-intensive situations.
+				[b]Note:[/b] In C#, [param property] must be in snake_case when referring to built-in Godot properties.
```

Comment:
Add a recommendation to use the cached `StringName` like we have in the `Object.set` documentation.

https://github.com/godotengine/godot/blob/ba54c34551d1bda5139515d74e7d614ccbc43cea/doc/classes/Object.xml#L888